tox.ini

```diff
@@ -7,4 +7,9 @@ commands =
     pip install -r requirements.txt
     pip install -r test-requirements.txt
     python -m unittest2 discover
-    robot --log {envname}_log.html --loglevel TRACE tests/integration_tests/test_suites/
+    robot \
+        --loglevel TRACE \
+        --log {toxworkdir}/{envname}_log.html  \
+        --report {toxworkdir}/{envname}_report.html \
+        --output {toxworkdir}/{envname}_output.xml \
+        tests/integration_tests/test_suites/
```

Comment:
Don't forget the empty line at the end of the file.
When adding another line, it will show as a diff against the empty line, and not against `tests/integration_tests/test_suites/` 
This is also why GitHub hows a big red icon for missing newlines at the end of the file.